W/wannier90/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "wannier90"
+version = v"3.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/wannier-developers/wannier90/archive/v3.1.0.tar.gz", "40651a9832eb93dec20a8360dd535262c261c34e13c41b6755fa6915c936b254")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/wannier90-*/
+
+shared="-shared"
+if [[ "${target}" == *-apple-* ]]; then
+    shared="-dynamiclib"
+fi
```

Comment:
I'm mildly sure that `-shared` works also for macOS libraries
```suggestion
```

Reply:
I'll remove that stuff completely then.